src/components/editor/editor-pane/autocompletion/code-block.ts

```diff
@@ -21,7 +21,7 @@ const codeBlockHint = (editor: Editor): Promise< Hints| null > => {
       }
       const term = searchResult[1]
       if (allSupportedLanguages.length === 0) {
-        allSupportedLanguages = hljs.listLanguages().concat('csv', 'flow', 'html', 'js', 'markmap', 'abc', 'graphviz', 'mermaid', 'vega-lite')
+        allSupportedLanguages = hljs.default.listLanguages().concat('csv', 'flow', 'html', 'js', 'markmap', 'abc', 'graphviz', 'mermaid', 'vega-lite')
```

Comment:
If we build custom language files anyway, this could be moved to there, couldn't it?
Also https://github.com/hedgedoc/react-client/blob/4cb60491b5cdfde1cd0f32a6a6b78f47b09b90bb/src/components/markdown-renderer/replace-components/highlighted-fence/highlighted-code/highlighted-code.tsx#L29-L38 could be changed, too?

Reply:
Good point. Done